rust/protocol/src/sealed_sender.rs

```diff
@@ -1194,6 +1569,12 @@ impl SealedSenderDecryptionResult {
     }
 }
 
+/// Decrypt a Sealed Sender message `ciphertext` in either the v1 or v2 format, and validate its
+/// sender certificate.
```

Comment:
", validate its sender certificate, and then decrypt the inner message payload". That's the most important part! :-)

Reply:
Thanks for clarifying this part!! Definitely a necessary part of the method description :)